restli-int-test-api/src/main/idl/com.linkedin.restli.examples.greetings.client.greeting.restspec.json

```diff
@@ -82,6 +82,107 @@
           "entity" : {
             "path" : "/greeting/subgreetings/{subgreetingsId}",
             "subresources" : [ {
+              "name" : "subgreetingsInsubgreetings",
+              "namespace" : "com.linkedin.restli.examples.greetings.client",
+              "path" : "/greeting/subgreetings/{subgreetingsId}/subgreetingsInsubgreetings",
+              "schema" : "com.linkedin.restli.examples.greetings.api.Greeting",
+              "doc" : "This resource represents a collection resource under a simple resource.\n\ngenerated from: com.linkedin.restli.examples.greetings.server.CollectionUnderCollectionResource",
+              "collection" : {
+                "identifier" : {
+                  "name" : "subgreetingsInsubgreetingsId",
+                  "type" : "long"
+                },
+                "supports" : [ "batch_create", "batch_get", "create", "delete", "get", "partial_update", "update" ],
+                "methods" : [ {
+                  "method" : "create"
+                }, {
+                  "method" : "get"
+                }, {
+                  "method" : "update"
+                }, {
+                  "method" : "partial_update"
+                }, {
+                  "method" : "delete"
+                }, {
+                  "method" : "batch_create"
+                }, {
+                  "method" : "batch_get"
+                } ],
+                "finders" : [ {
+                  "name" : "search",
+                  "parameters" : [ {
+                    "name" : "tone",
+                    "type" : "com.linkedin.restli.examples.greetings.api.Tone",
+                    "optional" : true
+                  }, {
+                    "name" : "complexQueryParam",
+                    "type" : "com.linkedin.restli.examples.greetings.api.Greeting",
+                    "optional" : true
+                  } ],
+                  "pagingSupported" : true
+                } ],
+                "actions" : [ {
+                  "name" : "exceptionTest"
+                }, {
+                  "name" : "purge",
+                  "returns" : "int"
+                } ],
+                "entity" : {
+                  "path" : "/greeting/subgreetings/{subgreetingsId}/subgreetingsInsubgreetings/{subgreetingsInsubgreetingsId}",
+                  "subresources" : [ {
+                    "name" : "subsubgreetingsInsubGreeting",
+                    "namespace" : "com.linkedin.restli.examples.greetings.client",
+                    "path" : "/greeting/subgreetings/{subgreetingsId}/subgreetingsInsubgreetings/{subgreetingsInsubgreetingsId}/subsubgreetingsInsubGreeting",
+                    "schema" : "com.linkedin.restli.examples.greetings.api.Greeting",
+                    "doc" : "This resource represents a collection resource under a simple resource.\n\ngenerated from: com.linkedin.restli.examples.greetings.server.ColectionUnderCollectionUnderCollectionResource",
```

Comment:
same here. 

I think two levels is good enough for tests. Do we really need this deep scenario tested? 
I don't think even the second level is going to be used in the real world.

Reply:
I am ok with not adding them, but I have tested these by checking out the result from this for simple keys, compound key and complex key.

Reply:
Since you've already added the resources, it is okay to keep these